cpp/benchmark/benchmark_packed.cpp

```diff
@@ -75,22 +78,23 @@ static void PackedRead(benchmark::State& st, arrow::fs::FileSystem* fs, const st
 
   // after writing, the column of large_str is in 0th file, and the last int64 columns are in 1st file
   std::vector<std::shared_ptr<arrow::Field>> fields = {
-      arrow::field("str", arrow::utf8()),
-      arrow::field("int32", arrow::int32()),
-      arrow::field("int64", arrow::int64()),
+      arrow::field("int", arrow::utf8()),
+      arrow::field("int64", arrow::int32()),
+      arrow::field("str", arrow::int64()),
   };
   auto schema = arrow::schema(fields);
 
   for (auto _ : st) {
     PackedRecordBatchReader pr(*fs, paths, schema, column_offsets, needed_columns, buffer_size);
-    auto r = pr.ToTable();
-    if (!r.ok()) {
-      st.SkipWithError(r.status().ToString());
-    }
-    auto table = r.ValueOrDie();
-    r = pr.Close();
-    if (!r.ok()) {
-      st.SkipWithError(r.status().ToString());
+    auto r = arrow::RecordBatch::MakeEmpty(schema);
+    SKIP_IF_NOT_OK(r.status(), st)
```

Comment:
Does it affect the benchmark result if we skip?

Reply:
Skip means abort the benchmark if there is error.